ports/rp2/mpconfigport.h

```diff
@@ -147,6 +147,8 @@
 #define MICROPY_SSL_MBEDTLS                     (1)
 #define MICROPY_PY_LWIP_SOCK_RAW                (MICROPY_PY_LWIP)
 
+#define MICROPY_HW_SOFT_TIMER_ALARM_NUM         (2)
```

Comment:
I would add a comment describing what this is for (it's not the number of alarms but rather the index).  Maybe call it `MICROPY_HW_SOFT_TIMER_ALARM_INDEX`?

Reply:
I agree with you about naming, but this is a pico-sdk convention (they have `PICO_TIME_DEFAULT_ALARM_POOL_HARDWARE_ALARM_NUM` and the argument is `hardware_alarm_num`, etc.)

Will add the comment :)